languageBase/A_LANG_CODE.json

```diff
@@ -1,44 +1,43 @@
 {
-    "Ovuli":" ",
-    "Please select your language":" ",
-    "Assamese":" as ",
-    "Bengali":" be ", 
-    "English":" en ",
-    "Gujarati":" gu ", 
-    "Hindi":" hi ", 
-    "Kannada":" kn ", 
-    "Kashmiri":" ks ", 
-    "Konkani":" ko ",
-    "Maithili":" ma ", 
-    "Malayalam":" ml ", 
-    "Manipuri":" mn ", 
-    "Marathi":" mr ", 
-    "Nepali":" ne ", 
-    "Oriya":" or ", 
-    "Punjabi":" pa ", 
-    "Sindhi":" sd ", 
-    "Tamil":" ta ", 
-    "Telugu":" te ",
-    "Urdu":" ur ",
+    "Ovuli": " ",
+    "Please select your language": " ",
+    "Assamese": " as ",
+    "Bengali": " be ",
+    "English": " en ",
+    "Gujarati": " gu ",
+    "Hindi": " hi ",
+    "Kannada": " kn ",
+    "Kashmiri": " ks ",
+    "Konkani": " ko ",
+    "Maithili": " ma ",
+    "Malayalam": " ml ",
+    "Manipuri": " mn ",
+    "Marathi": " mr ",
+    "Nepali": " ne ",
+    "Oriya": " or ",
+    "Punjabi": " pa ",
+    "Sindhi": " sd ",
+    "Tamil": " ta ",
+    "Telugu": " te ",
+    "Urdu": " ur ",
     "How can we call you?": " ",
     "How Long is your is cycle?": " ",
     "When was your last period?": " ",
-    "Do you know your cycle length?":" ",
+    "Do you know your cycle length?": " ",
     "Yes": " ",
     "No": " ",
     "When was your second last period?": " ",
     "When was your third last period?": " ",
-    "Continue":" ",
+    "Continue": " ",
     "History": " ",
     "Home": " ",
     "Settings": " ",
     "Language": " ",
     "Cycle length": " ",
     "Last Period": " ",
     "Alert": " ",
-    "Please enter your Name to proceed" : " ",
+    "Please enter your Name to proceed": " ",
     "OK": " ",
     "Finish": " ",
     "RESET": " "
```

Comment:
I think this is not needed. Remove this code.

Reply:
Okay